File: Java/13-pattern-increasing-number-triangle.java
import java.util.Scanner;

public class Solution {
    public static void nNumberTriangle(int n) {
        // Write your code here
        int num = 1;
        for(int i = 1; i<=n; i++){
            for(int j = 1; j<=i; j++){
                System.out.print(num + " ");
                num++;
            }
            System.out.println();
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        nNumberTriangle(n);
        scanner.close();
    }
}

File: Python/13-pattern-increasing-number-triangle.py
def nNumberTriangle(n):
    # Write your code here
    num = 1
    for i in range(1, n + 1):
        for j in range(1, i + 1):
            print(num, end=" ")
            num += 1
        print()

def main():
    n = int(input())
    nNumberTriangle(n)

if __name__ == "__main__":
    main()
